vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,72 @@
+"""
+renders 3d point for 2d surfaces
+"""
+
+import math, copy
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+AXIS_X = "x"
+AXIS_Y = "y"
+AXIS_Z = "z"
+
+
+def convert_to_2d(x: int, y: int, z: int, scale: int, distance: int) -> tuple:
```

Comment:
Could x be a float instead of an int?  Please add a doctest for that.

The return type should be `tuple[float, float]` or `tuple[int, int]`.

Reply:
how do i make that `tuple[float, float]`, what should i write?

Reply:
ok i got it done